core/src/main/java/org/jboss/sbomer/core/features/sbom/utils/SbomUtils.java

```diff
@@ -412,6 +429,26 @@ private static void addLicenseEvidence(Component component, List<LicenseInfo> li
         component.setEvidence(evidence);
     }
 
+    public static List<Hash> getHashesFromAnalyzedDistroribution(AnalyzedDistribution analyzedDistribution) {
+        List<Hash> hashes = new ArrayList<>();
+
+        if (analyzedDistribution == null) {
+            return hashes;
+        }
```

Comment:
There is a spelling error in the method name here.

Also, you use `Optional.ofNullable` earlier, but this method currently can't return `null`. I think returning an empty list is easier, but if you want to return `null`, then you could move this first `null` check here up to the first line of the method, and also return `null` below if  `hashes` is empty.